vcd/datasource_vcd_nsxt_alb_settings_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build nsxt || alb || ALL || functional
+// +build nsxt alb ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+// TestAccVcdNsxtAlbSettingsDS assumes that NSX-T ALB is not configured and General Settings shows "Inactive"
+func TestAccVcdNsxtAlbSettingsDS(t *testing.T) {
+	preTestChecks(t)
+
+	// This test requires access to the vCD before filling templates
+	// Thus it won't run in the short test
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	vcdClient := createTemporaryVCDConnection()
+	if vcdClient.Client.APIVCDMaxVersionIs("< 35.0") {
+		t.Skip(t.Name() + " requires at least API v35.0 (vCD 10.2+)")
```

Comment:
Officially, capitalization is "VCD" after the name change to VMware Cloud Director. I see you are using "VCD" below in this PR already :)

Reply:
Yes. This is actually something that gets carried over many places as we usually copy this boilerplate. I changed all occurrences in my PR, but might need to tackle other leftovers in future.